doc/python_reference.rst

```diff
@@ -698,6 +698,7 @@ Inverse Solutions
    rap_music
    tf_dics
    tf_lcmv
+   make_resolution_matrix
```

Comment:
I think this is the first time we'll have an equivalent naming (albeit in different namespaces), as we'll have `mne.minimum_norm.make_resolution_matrix` and `mne.beamformer.make_resolution_matrix`. I guess it's okay but maybe it makes more sense to do `mne.beamformer.make_lcmv_resolution_matrix` and `mne.minimum_norm.make_inverse_resolution_matrix`. It's closer to how the namings work in those modules anyway with the `_lcmv` and `_inverse` in the names.